.github/workflows/tests.yml

```diff
@@ -70,6 +70,6 @@ jobs:
           PYTHON_VERSION: ${{ matrix.python-version }}
           PLONE_VERSION: ${{ matrix.plone-version }}
 
-      # test sphinx warnings
-#      - name: sphinx
-#        run: if [ "${{ matrix.plone-version }}" == "5.2" ] && [ ${{ matrix.python-version }} == '3.7' ]; then ./test-no-sphinx-warnings; fi
+      # test for broken links
+      - name: linkcheck
+        run: if [ "${{ matrix.plone-version }}" == "6.0" ] && [ ${{ matrix.python-version }} == '3.9' ]; then make docs-linkcheckbroken; fi
```

Comment:
Although it is slightly out of scope, let's clean up the other sphinx task by removing the comment and bumping plone and python versions to align with the latest:

https://github.com/plone/plone.restapi/pull/1422/files/dfb88d317460c824fa9a99abbcc9320e2e40bc2b..553a4bbde7303296586ca7b67d8c4f4cfa538836#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fR59-R60